dotCMS/src/main/java/com/dotcms/rest/api/v1/experiments/ExperimentsResource.java

```diff
@@ -0,0 +1,159 @@
+package com.dotcms.rest.api.v1.experiments;
+
+import com.dotcms.experiments.business.ExperimentsAPI;
+import com.dotcms.experiments.model.Experiment;
+import com.dotcms.rest.InitDataObject;
+import com.dotcms.rest.PATCH;
+import com.dotcms.rest.ResponseEntityView;
+import com.dotcms.rest.WebResource;
+import com.dotcms.rest.annotation.NoCache;
+import com.dotcms.rest.exception.NotFoundException;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.liferay.portal.model.User;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.Collections;
+import java.util.Optional;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import org.glassfish.jersey.server.JSONP;
+import org.jetbrains.annotations.NotNull;
+
+@Path("/v1/experiments")
+@Tag(name = "Experiment")
+public class ExperimentsResource {
+
+    private final WebResource webResource;
+    private final ExperimentsAPI experimentsAPI;
+
+    public ExperimentsResource() {
+        webResource =  new WebResource();
+        experimentsAPI = APILocator.getExperimentsAPI();
+    }
+
+    @POST
+    @JSONP
+    @NoCache
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    public ResponseEntityExperimentView create(@Context final HttpServletRequest request,
+            @Context final HttpServletResponse response,
+            final ExperimentForm experimentForm) throws DotDataException, DotSecurityException {
```

Comment:
if you do not call the validate, the API is not calling it by you

Reply:
Add swagger annotations
and log the input

Reply:
Done